onboarding/README.md

```diff
@@ -91,3 +92,10 @@ If you are using your GitHub profile photo instead of uploading a photo skip thi
 
 1. Commit your changes.
 2. From GitHub create a pull request and assign it to a member of the Marketing team to review.
+
+
+## Configure your email signature
+
+- Select and copy the template signature <a href="email-signature.html" target="_blank">here</a> 
+- Paste the template in the signature settings of your email
+- Change the texts with your infos (don't forget to change both, text displays and links)
```

Comment:
Maybe one final step to send a test email to someone to confirm that your signature appears correctly?

Reply:
Good idea @geekgonecrazy! I'll add it on the steps!